doc/api/fs.md

```diff
@@ -2697,10 +2697,19 @@ changes:
 * `callback` {Function}
   * `err` {Error}
 
-Asynchronously rename file at `oldPath` to the pathname provided
+Asynchronously rename file or directory at `oldPath` to the pathname provided
 as `newPath`. In the case that `newPath` already exists, it will
-be overwritten. No arguments other than a possible exception are
-given to the completion callback.
+be overwritten. `fs.rename` can throw the below error:
+
+- On **Linux** when renaming directory exists and is NOT empty, it throws a
```

Comment:
1. Not sure if `renaming` is a right form here and below. Let's see what native speakers think.
2. `` a `ENOTEMPTY` `` - > `` an `ENOTEMPTY` `` (as well as below).

Reply:
Its not clear or grammatical, call it after the name of the arg: "if `newPath` exists"... and likewise in other places.